ghcide/src/Development/IDE/Core/Shake.hs

```diff
@@ -706,6 +708,11 @@ shakeRestart recorder IdeState{..} reason acts =
         -- See https://github.com/haskell/ghcide/issues/79
         (\() -> do
           (,()) <$> newSession recorder shakeExtras shakeDb acts reason)
+    where
+        errorAfter :: Seconds -> Recorder (WithPriority Log) -> IO () -> IO ()
```

Comment:
From the name I was expecting this to actually throw an error, but it just logs, surprisingly! Maybe rename.

Reply:
renamed, thanks